### ISSUE TITLE
Azteccode generate fails with version 3.5.0 and specific payload

### ISSUE THREAD

**Issue by tned73** [adamchainz/treepoem#264]
With the following test code running with Treepoem version 3.5.0 we get a error : "GPL Ghostscript 9.27: Unrecoverable error, exit code 1"
also tried it with Ghostscript 9.53.3, same issue, downgrading to Treepoem 3.4.0 solved the problem.

With this specific data it will always fail, other data (like `bytes(range(25))` ) generates an image, without a problem

`
treepoem.generate_barcode(
        barcode_type="azteccode",
        data=bytes(range(256)),
        options={"layers": 17, "format": "full"},
     )
`

**Comment by adamchainz** [comment]
Full stack trace:

```
$ python
Python 3.8.5 (default, Sep  3 2020, 10:31:10)
[Clang 11.0.0 (clang-1100.0.20.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import treepoem
>>> treepoem.generate_barcode( barcode_type="azteccode", data=bytes(range(256)), options={"layers": 17, "format": "full"}, )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/chainz/Documents/Projects/treepoem/src/treepoem/__init__.py", line 148, in generate_barcode
    bbox_lines = _get_bbox(code)
  File "/Users/chainz/Documents/Projects/treepoem/src/treepoem/__init__.py", line 100, in _get_bbox
    raise TreepoemError(err_output)
treepoem.TreepoemError: GPL Ghostscript 9.52: Unrecoverable error, exit code 1
```

It seems the maximum is `bytes(range(48))` - `bytes(range(49))` fails.

This also happens with the latest BWIPP, 2020-10-02 - https://github.com/bwipp/postscriptbarcode/releases 

The latest version it didn't happen with is 2020-09-11 .

I'm reporting upstream on BWIPP's issue tracker.

**Comment by adamchainz** [comment]
Issue: https://github.com/bwipp/postscriptbarcode/issues/132 .

**Comment by adamchainz** [comment]
Fixed upstream, fix merged in #265, released in version 3.6.0. Enjoy!
